docs/DeveloperGuide.adoc

```diff
@@ -316,14 +316,158 @@ _{More to be added}_
 (For all use cases below, the *System* is the `AddressBook` and the *Actor* is the `user`, unless specified otherwise)
 
 [discrete]
-=== Use case: Delete person
+=== Use case: Viewing a page
 
 *MSS*
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1.  User requests to view a different page
+2. AddressBook layout changes to show the new page
+
++
+Use case ends.
+
+*Extensions*
+
+* 1a.  The requested page is invalid.
++
+[none]
+** 1a1.  AddressBook shows an error message.
++
+Use case resumes at step 1.
+
+[discrete]
+=== Use case: Viewing the help menu
+
+*MSS*
+
+1.  User requests to view the help menu
+2. AddressBook shows the list of commands
++
+Use case ends.
+
+[discrete]
+=== Use case: Change default view
+
+*MSS*
+
+1.  User requests to change the default view to a specified page
+2. AddressBook changes the default view
+3. AddressBook shows the default view
++
+Use case ends.
+
+*Extensions*
+
+* 1a. The given page is invalid.
++
+[none]
+** 1a1. AddressBook shows an error message.
++
+Use case resumes at step 1.
+
+[discrete]
+=== Use case: Change the hourly pay rate
+
+*MSS*
+
+1.  User requests to change the hourly pay rate to a specified amount
+2. AddressBook changes the pay rate
+3. AddressBook shows an edited TSS claims page the total pay adjusted to reflect the new pay rate
```

Comment:
You put full stops some places but don't in other places. Can make it more uniform?